### PR TITLE
Some small adjustments during code review

### PR DESCRIPTION
**Pull request by aarmey**
Ok, this was going to be small, but required some more extensive changes to get everything working. A summary:
- I committed Fig. 8 to speed up the build a bit.
- The gamma distribution fitting is unified under a single function. This might be a tad slower under certain circumstances, but guarantees that the estimator is more extensively tested.
- Because the fitting process doesn't have very many unknowns, there's little benefit from jax. I was having trouble with the JIT compiler, so the Numba + diffing option came out better.
- I did some cleanup of the unit tests. It would be helpful to go through these one day because I'm not sure that all are effectively testing for anything. A couple are extremely slow.

### CODE REVIEW

File: lineage/states/StateDistributionGamma.py [hunk 1]
```diff
@@ -202,7 +202,7 @@ def atonce_estimator(all_tHMMobj: list, x_list: list, gammas_list: list, phase:
     x_data = [np.array(x) for x in x_list]
 
     # getting the observations as individual lists
-    bern_obs = [x[:, 0].astype('bool') for x in x_data]
+    bern_obs = [x[:, 0] for x in x_data]
```
[the remark this answers is not in the document]
Comment (Farnazmdi) [reply lineage/states/StateDistributionGamma.py:205]: oh.. you are right.. I think the `.astype('bool')` should be applied in line 219 that we are making g_mask..

[the remark this answers is not in the document]
Comment (Farnazmdi) [reply lineage/states/StateDistributionGamma.py:205]: I can make that change if it is okay.

[the remark this answers is not in the document]
Comment (aarmey) [reply lineage/states/StateDistributionGamma.py:205]: Please do.

**Comment by aarmey** [comment]
@Farnazmdi please look through this and the results once built. If you're satisfied with it all then we can merge.

**Review by Farnazmdi — APPROVED** [review]
wow.. thank you so much. I really appreciate it. I checked the figures and they look good. I will go through the tests after finishing up the response to reviewers if that is alright.

**Comment by Farnazmdi** [comment]
Thank you again.. merging..

**Pull request merged**